File: JavaScript/Object/Copy.js
var a = {a: 1, b: 2};

var b = a;
b.a = 3; 
document.body.innerHTML += a.a; //imprime 3

function change(objeto) {
    objeto.a = 3;
}

change(a);
document.body.innerHTML += a.a; //imprime 3

function copy(objeto) {
    return objeto;
}

var b = copy(a);
b.a = 3;
document.body.innerHTML += a.a; //imprime 3

//https://pt.stackoverflow.com/q/125816/101

File: C/Array/Copy.c
memcpy(e1, e2, sizeof(events) * 100);

copy(begin(e1), end(e1), begin(e2));

for (auto i = 0; i < 100; i++)  e2[i] = e1[i];

//https://pt.stackoverflow.com/q/322684/101
